<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QWidget" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="sendLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>620</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back/qss/psblack/paperplane.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="serverIPLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>服务器地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverIPLineEdit">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLineEdit">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back/qss/psblack/icon_4.png);</string>
   </property>
   <property name="text">
    <string>进入聊天室</string>
   </property>
  </widget>
  <widget class="QLabel" name="IPLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditClient">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="listViewOnline">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>379</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>280</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前选中：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditObject">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>320</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back/qss/psblack/customer.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在线成员</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>709</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFont">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改字体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonColor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改颜色</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditColorShow">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>650</y>
     <width>21</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>sendLineEdit</zorder>
  <zorder>sendBtn</zorder>
  <zorder>userNameLabel</zorder>
  <zorder>serverIPLabel</zorder>
  <zorder>serverIPLineEdit</zorder>
  <zorder>portLabel</zorder>
  <zorder>portLineEdit</zorder>
  <zorder>enterBtn</zorder>
  <zorder>IPLabel</zorder>
  <zorder>textEditClient</zorder>
  <zorder>listViewOnline</zorder>
  <zorder>label</zorder>
  <zorder>textEditObject</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButtonFont</zorder>
  <zorder>pushButtonColor</zorder>
  <zorder>lineEditColorShow</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
